### DISCUSSION TITLE
16 giugno 2022

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/mdp2223#5] [Q&A] [answered]
In questa discussion a tema la traccia di esame del 16 giugno 2022 discuteremo di eventuali proposte di soluzione all'esame in questione

**Comment by Gio0203c** [comment, marked as answer]
[16_06_2022 - COMPLETO.pdf](https://github.com/sapienzastudentsnetwork/mdp2223/files/12988514/16_06_2022.-.COMPLETO.pdf)

**Comment by Frisbon** [comment]
É un dupplicato di [17.05...](https://github.com/sapienzastudentsnetwork/mdp2223/discussions/6)

**Reply by Gio0203c** [reply]
Lo so, però avendoli fatti in ordine cronologico non potevo saperlo. Qui ci sono meno esercizi perché prima si preparava anche il progetto
